src/components/LastDigitStats.jsx

```diff
@@ -3,19 +3,20 @@ import { connect } from '../store/Connect';
 import '../../sass/components/_last-digits.scss';
 
 const Bar = ({ x, bar }) => (
-    <div className={`cq-bar ${bar.cName}`} style={{ height: bar.height, left : (x * 35), '--bar-gradiant': `${bar.gradiantLine}%` }}>
+    <div className={`cq-bar ${bar.cName}`} style={{ height: `${bar.height * 5}%`, left : (x * 35) }}>
```

Comment:
@MahboobehMohammadi Would you plz put each attribute in a separate line? it makes the code more readable.
Also, It's more readable to use variables for calculated values in then use the variables in the JSX tags.

Reply:
@morteza-binary we usually let eslint handle this kindda things. if eslint don't show errors means it's readable i think. 
i myself don't like separating  every props in single line. harder for me to read them.

Reply:
@MahboobehMohammadi Unfortunately there isn't the related rules enabled in our project so eslint doesn't manage it.

Reply:
@morteza-binary so we should add releated rules if there are there and have eslint to handle these changes, so code looks like the same everywhere.

Reply:
@MahboobehMohammadi Unfortunately, we can't add it the rule now, because it causes a lot of errors. I'm planning to add a card for refactoring the project style and rules in the further next sprints. But now we can follow it and instead of eslint.